docs/assets/2.2-2.3-cleanup-orphaned-svcat-resources.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+echo "Deleting Application Connectivity related resources"
+kubectl delete crd applicationmappings.applicationconnector.kyma-project.io
+kubectl delete crd eventactivations.applicationconnector.kyma-project.io
+kubectl delete crd tokenrequests.applicationconnector.kyma-project.io 
+
+for APP in $(kubectl get app -oname | cut -d '/' -f 2)
+do
+  kubectl delete svc,deploy -l app="${APP}-application-gateway" -n kyma-integration
+done
+
+echo "Deleting rafter"
+
+kubectl delete crd assetgroups.rafter.kyma-project.io
+kubectl delete crd assets.rafter.kyma-project.io
+kubectl delete crd buckets.rafter.kyma-project.io
+kubectl delete crd clusterassetgroups.rafter.kyma-project.io
+kubectl delete crd clusterassets.rafter.kyma-project.io
+kubectl delete crd clusterbuckets.rafter.kyma-project.io
+
+kubectl -n kyma-system delete service rafter-asyncapi-service
+kubectl -n kyma-system delete service rafter-controller-manager
+kubectl -n kyma-system delete service rafter-front-matter-service
+kubectl -n kyma-system delete service rafter-minio
+kubectl -n kyma-system delete service rafter-upload-service
+
+kubectl -n kyma-system delete deployment rafter-asyncapi-svc
+kubectl -n kyma-system delete deployment rafter-ctrl-mngr
+kubectl -n kyma-system delete deployment rafter-front-matter-svc
+kubectl -n kyma-system delete deployment rafter-minio
+kubectl -n kyma-system delete deployment rafter-upload-svc
+
+echo "Deleting Service Catalog"
+
+kubectl delete crd addonsconfigurations.addons.kyma-project.io
+kubectl delete crd clusteraddonsconfigurations.addons.kyma-project.io
+kubectl delete crd podpresets.settings.svcat.k8s.io
+kubectl delete crd clusterservicebrokers.servicecatalog.k8s.io
+kubectl delete crd clusterserviceclasses.servicecatalog.k8s.io
+kubectl delete crd clusterserviceplans.servicecatalog.k8s.io
+kubectl delete crd servicebindings.servicecatalog.k8s.io
+kubectl delete crd servicebindingusages.servicecatalog.kyma-project.io
```

Comment:
I see sc-migration reconciler did not remove instances of SBU crd, we need to remove all finalizers from instances before removing SBU CRD

Reply:
I think that was potentially some transient error, the SBUs eventually got removed and it's implemented as part of [scmigration reconciler](https://github.com/kyma-incubator/reconciler/blob/5ff701d373650dc494cabc588bd902277b187f6a/pkg/reconciler/instances/scmigration/sc_removal.go#L292-L296) along with [handling of finalizer and owner references](https://github.com/kyma-incubator/reconciler/blob/5ff701d373650dc494cabc588bd902277b187f6a/pkg/reconciler/instances/scmigration/sc_removal.go#L62)